packages/fx-core/templates/core/v3Migration/js.ts.app.yml

```diff
@@ -124,6 +127,7 @@ deploy:
     with:
       workingDirectory: api
       args: build extensions.csproj -o bin --ignore-failed-sources
+      execPath: $\{{ {{~dotnetPath~}} }}
```

Comment:
Let's add a comment like: "Use dotnet installed by prerequisite/install action" to help users better understand this parameter

Reply:
added